homework7.py

```diff
@@ -0,0 +1,55 @@
+"""just a dockstring"""
+
+# Zadanie1
+number = "2310"
```

Comment:
pls do not hardcode the value, generate it with random library https://docs.python.org/3/library/random.html